### ISSUE TITLE
"VisualStudio" build target fails

### ISSUE THREAD

**Issue by Wortex17** [JetBrains/resharper-rider-plugin#23]
I am not able to succesfully run the "VisualStudio" build target. I get the following errors instead:
```
Downloading from https://download.jetbrains.com/resharper/ReSharperUltimate.2019.2.2/JetBrains.ReSharperUltimate.2019.2.2.exe
Installing experimental hive
> F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\build\installer\JetBrains.ReSharperUltimate.2019.2.2.exe /VsVersion=16.0 /SpecificProductNames=ReSharper /Hive=MyAwesomePlugin /Silent=True
3176
> dotnet msbuild /t:Restore;Rebuild;Pack F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\MyAwesomePlugin.sln /p:PackageVersion=1337.0.0 /p:PackageOutputPath="F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\output"
3164
Microsoft (R)-Build-Engine, Version 16.2.32702+c4012a063 für .NET Core
Copyright (C) Microsoft Corporation. Alle Rechte vorbehalten.

  Wiederherstellung in "338,33 ms" für "F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\src\dotnet\ReSharperPlugin.MyAwesomePlugin.Tests\ReSharperPlugin.MyAwesomePlugin.Tests.csproj" abgeschlossen.
  Wiederherstellung in "338,33 ms" für "F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\src\dotnet\ReSharperPlugin.MyAwesomePlugin\ReSharperPlugin.MyAwesomePlugin.csproj" abgeschlossen.
  Wiederherstellung in "338,33 ms" für "F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\src\dotnet\ReSharperPlugin.MyAwesomePlugin\ReSharperPlugin.MyAwesomePlugin.Rider.csproj" abgeschlossen.
C:\Program Files\dotnet\sdk\2.2.402\Microsoft.Common.CurrentVersion.targets(2106,5): warning MSB3277: Es wurden Konflikte zwischen verschiedenen Versionen von "System.Collections.Immutable" gefunden, die nicht gelöst werden konnten. Diese Verweisk
onflikte werden im Buildprotokoll aufgelistet, wenn die Protokollausführlichkeit auf "Ausführlich" festgelegt ist. [F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\src\dotnet\ReSharperPlugin.MyAwesomePlugin\ReSharperPlugin.MyAwesomePlu
gin.csproj]
  ReSharperPlugin.MyAwesomePlugin -> F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\src\dotnet\ReSharperPlugin.MyAwesomePlugin\bin\ReSharperPlugin.MyAwesomePlugin\Debug\ReSharperPlugin.MyAwesomePlugin.dll
  ReSharperPlugin.MyAwesomePlugin.Rider -> F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\src\dotnet\ReSharperPlugin.MyAwesomePlugin\bin\ReSharperPlugin.MyAwesomePlugin.Rider\Debug\ReSharperPlugin.MyAwesomePlugin.dll
C:\Program Files\dotnet\sdk\2.2.402\Microsoft.Common.CurrentVersion.targets(2106,5): warning MSB3277: Es wurden Konflikte zwischen verschiedenen Versionen von "System.Collections.Immutable" gefunden, die nicht gelöst werden konnten. Diese Verweisk
onflikte werden im Buildprotokoll aufgelistet, wenn die Protokollausführlichkeit auf "Ausführlich" festgelegt ist. [F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\src\dotnet\ReSharperPlugin.MyAwesomePlugin.Tests\ReSharperPlugin.MyAwes
omePlugin.Tests.csproj]
  ReSharperPlugin.MyAwesomePlugin.Tests -> F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\src\dotnet\ReSharperPlugin.MyAwesomePlugin.Tests\bin\ReSharperPlugin.MyAwesomePlugin.Tests\Debug\ReSharperPlugin.MyAwesomePlugin.Tests.dll
  Das Paket "F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\output\ReSharperPlugin.MyAwesomePlugin.1337.0.0.nupkg" wurde erfolgreich erstellt.
> F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\tools\nuget.exe install ReSharperPlugin.MyAwesomePlugin -OutputDirectory C:\Users\drago\AppData\Local\JetBrains\plugins -Source F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlug
in\output -DependencyVersion Ignore
3184
Feeds used:
  F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\output

Installing package 'ReSharperPlugin.MyAwesomePlugin' to 'C:\Users\drago\AppData\Local\JetBrains\plugins'.
Resolved actions to install package 'ReSharperPlugin.MyAwesomePlugin.1337.0.0'
Retrieving package 'ReSharperPlugin.MyAwesomePlugin 1337.0.0' from 'F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\output'.
Adding package 'ReSharperPlugin.MyAwesomePlugin.1337.0.0' to folder 'C:\Users\drago\AppData\Local\JetBrains\plugins'
Added package 'ReSharperPlugin.MyAwesomePlugin.1337.0.0' to folder 'C:\Users\drago\AppData\Local\JetBrains\plugins'
Successfully installed 'ReSharperPlugin.MyAwesomePlugin 1337.0.0' to C:\Users\drago\AppData\Local\JetBrains\plugins
Executing nuget actions took 71,65 ms
Re-installing experimental hive
> F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\build\installer\JetBrains.ReSharperUltimate.2019.2.2.exe /VsVersion=16.0 /SpecificProductNames=ReSharper /Hive=MyAwesomePlugin /Silent=True
1088
> dotnet msbuild /t:Restore;Rebuild F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\MyAwesomePlugin.sln
3160
Microsoft (R)-Build-Engine, Version 16.2.32702+c4012a063 für .NET Core
Copyright (C) Microsoft Corporation. Alle Rechte vorbehalten.

  Wiederherstellung in "296,8 ms" für "F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\src\dotnet\ReSharperPlugin.MyAwesomePlugin\ReSharperPlugin.MyAwesomePlugin.csproj" abgeschlossen.
  Wiederherstellung in "296,79 ms" für "F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\src\dotnet\ReSharperPlugin.MyAwesomePlugin.Tests\ReSharperPlugin.MyAwesomePlugin.Tests.csproj" abgeschlossen.
  Wiederherstellung in "296,8 ms" für "F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\src\dotnet\ReSharperPlugin.MyAwesomePlugin\ReSharperPlugin.MyAwesomePlugin.Rider.csproj" abgeschlossen.
  ReSharperPlugin.MyAwesomePlugin.Rider -> F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\src\dotnet\ReSharperPlugin.MyAwesomePlugin\bin\ReSharperPlugin.MyAwesomePlugin.Rider\Debug\ReSharperPlugin.MyAwesomePlugin.dll
C:\Program Files\dotnet\sdk\2.2.402\Microsoft.Common.CurrentVersion.targets(2106,5): warning MSB3277: Es wurden Konflikte zwischen verschiedenen Versionen von "System.Collections.Immutable" gefunden, die nicht gelöst werden konnten. Diese Verweisk
onflikte werden im Buildprotokoll aufgelistet, wenn die Protokollausführlichkeit auf "Ausführlich" festgelegt ist. [F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\src\dotnet\ReSharperPlugin.MyAwesomePlugin\ReSharperPlugin.MyAwesomePlu
gin.csproj]
  ReSharperPlugin.MyAwesomePlugin -> F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\src\dotnet\ReSharperPlugin.MyAwesomePlugin\bin\ReSharperPlugin.MyAwesomePlugin\Debug\ReSharperPlugin.MyAwesomePlugin.dll
C:\Users\drago\.nuget\packages\jetbrains.platform.sdk\192.0.20190807.143715\DotFiles\JetBrains.CommonSdk.targets(24,5): error MSB4018: Unerwarteter Fehler bei der InstalledProductsDiscoveryTask-Aufgabe. [F:\DWPersonal\Projekte\dws\Infrablack\GCX\X
\MyAwesomePlugin\src\dotnet\ReSharperPlugin.MyAwesomePlugin\ReSharperPlugin.MyAwesomePlugin.csproj]
C:\Users\drago\.nuget\packages\jetbrains.platform.sdk\192.0.20190807.143715\DotFiles\JetBrains.CommonSdk.targets(24,5): error MSB4018: System.MissingMethodException: Method not found: 'System.Runtime.Remoting.ObjectHandle System.Activator.CreateIn
stance(System.String, System.String)'. [F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\src\dotnet\ReSharperPlugin.MyAwesomePlugin\ReSharperPlugin.MyAwesomePlugin.csproj]
C:\Users\drago\.nuget\packages\jetbrains.platform.sdk\192.0.20190807.143715\DotFiles\JetBrains.CommonSdk.targets(24,5): error MSB4018:    at JetBrains.Platform.SDK.Helper.InstalledProductsDiscoveryTask.Execute() [F:\DWPersonal\Projekte\dws\Infrabl
ack\GCX\X\MyAwesomePlugin\src\dotnet\ReSharperPlugin.MyAwesomePlugin\ReSharperPlugin.MyAwesomePlugin.csproj]
C:\Users\drago\.nuget\packages\jetbrains.platform.sdk\192.0.20190807.143715\DotFiles\JetBrains.CommonSdk.targets(24,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute() [F:\DWPerso
nal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\src\dotnet\ReSharperPlugin.MyAwesomePlugin\ReSharperPlugin.MyAwesomePlugin.csproj]
C:\Users\drago\.nuget\packages\jetbrains.platform.sdk\192.0.20190807.143715\DotFiles\JetBrains.CommonSdk.targets(24,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.ExecuteInstantiatedTask(ITaskExecutionHost taskExecutionHost, TaskLogg
ingContext taskLoggingContext, TaskHost taskHost, ItemBucket bucket, TaskExecutionMode howToExecuteTask) [F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\src\dotnet\ReSharperPlugin.MyAwesomePlugin\ReSharperPlugin.MyAwesomePlugin.csproj
]
'dotnet msbuild /t:Restore;Rebuild F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\MyAwesomePlugin.sln' failed with exit code 1, valid exit codes: 0
In F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\settings.ps1:33 Zeichen:9
+         throw "'$Executable $Arguments' failed with exit code $($rc.E ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: ('dotnet msbuild...d exit codes: 0:String) [], RuntimeException
    + FullyQualifiedErrorId : 'dotnet msbuild /t:Restore;Rebuild F:\DWPersonal\Projekte\dws\Infrablack\GCX\X\MyAwesomePlugin\MyAwesomePlugin.sln' failed with exit code 1, valid exit codes: 0
```

The actual error starts at 
```
C:\Users\drago\.nuget\packages\jetbrains.platform.sdk\192.0.20190807.143715\DotFiles\JetBrains.CommonSdk.targets(24,5): error MSB4018: Unerwarteter Fehler bei der InstalledProductsDiscoveryTask-Aufgabe. [F:\DWPersonal\Projekte\dws\Infrablack\GCX\X
\MyAwesomePlugin\src\dotnet\ReSharperPlugin.MyAwesomePlugin\ReSharperPlugin.MyAwesomePlugin.csproj]
```

Can someone verify for me that the "VisualStudio" target should, indeed, currently work?

Please note that
* I originaly also ran into the max-pathlength (#6) but I enabled pathlength support on my win10 OS. The paths have been shortened in the pasted log above for readability.
* My system language is german, which means some of the compiler messages are in german (sorry, I'm sad about that myself) - I'd be glad to translate on request
* I had to copy vswhere.exe from an older revision first, because of #22
* I have both, VS 2017 and VS 2019 installed, though I am not sure which one the script is trying to start (looking at the picked Resharper version, I suspect 2019)

**Comment by matkoch** [comment]
Can you please retry to create a project from template (including to copy vswhere.exe) ? I want to verify that it's not the path length. So something directly under `C:\code\xxxx`.

**Comment by matkoch** [comment]
We found the issue. Will push a new release very soon.

**Comment by matkoch** [comment]
@Wortex17 I just released 0.14.0. Please feel free to try.

**Comment by Wortex17** [comment]
Ic an verify that it builds and runs with the 0.14.0 template
